File: dashboard/src/main/java/com/pnfe/dashboard/dto/Account.java
package com.pnfe.dashboard.dto;

import io.swagger.annotations.ApiModelProperty;
import lombok.Data;

import java.util.List;

@Data
public class Account {
    @ApiModelProperty(required = true, value = "Идентификатор счета")
    private String accountId;
    @ApiModelProperty(required = true, value = "Реквизиты счета")
    private Requisite requisites;
    @ApiModelProperty(required = true, value = "Перечень карт к счету")
    private List<CardView> cards;
}

File: android/app/src/main/java/com/parcelnotfoundexception/javahack/domain/model/Account.kt
package com.parcelnotfoundexception.javahack.domain.model

import com.google.gson.annotations.SerializedName

data class Account(
    @SerializedName("accountId")
    val id: String,
    val cards: List<Card>,
    @SerializedName("requisites")
    val requisitesModel: RequisitesModel
)